docs/features/settings.rst

```diff
@@ -134,9 +134,9 @@ changes a setting, and will receive both the old value and the new value.
 
 .. code-block:: text
 
-   (Cmd) set --long | grep sunny
+   (Cmd) set -v | grep sunny
```

Comment:
I'm thinking we should use the long option name here (--verbose) to be more explicit in the documentation.
@tleonhardt Thoughts?

Reply:
Yeah, I agree that the full option name would be better. I'll go ahead and make the change

I also noticed that there were some references to `with_argparser_and_unknown_args` that I think should be `with_argparser(parser, with_unknown_args=True)`

Reply:
@kmvanbrunt  Yeah I like the `--verbose` because it makes it very clear what the intent is.

@KyleKing Good point about the outdated `with_argparser_and_unknown_args` docs.  That is an older form that is now deprecated.  You are correct that it should be `with_argparser(parser, with_unknown_args=True)`.  If you have the time and inclination, please submit another PR to fix that ;-)